src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -71,7 +71,6 @@ public abstract class AnalyzeOptionsBase : CommonOptionsBase
         public IEnumerable<string> PluginFilePaths { get; set; }
 
         [Option(
-            'i',
```

Comment:
>', [](start = 14, length = 2)

This short-cut interferes with 'inline'. This is a breaking change and needs to be mentioned in the release notes, will add.